### ISSUE TITLE
Copy annotations from one task to another

### ISSUE THREAD

**Issue by davodogster** [openvinotoolkit/cvat#3413]
Here I envision the Track (not shape) Polygon feature, but except tracking the polygon across images within the same task, The user is able to track/copy the polygon to another task with images of similar objects.
Are you able to make this possible?

Cheers,
Sam

**Comment by bsekachev** [comment]
Yes, the system clipboard can be used to improve copy/paste feature. 
Let us know if somebody can help us with the implementation.

**Comment by davodogster** [comment]
Hi @bsekachev has there been any progress? Cheers, Sam

**Comment by bsekachev** [comment]
@davodogster

We are not working on this feature for now.

**Comment by nmanovic** [comment]
OK. Different tasks can have similar objects (e.g. ignore regions). Thus it is a good idea to copy them from one task to another.

**Comment by davodogster** [comment]
@nmanovic Hi is there any progress on this? Cheers!! Sam
